pytorch_pfn_extras/training/manager.py

```diff
@@ -463,8 +463,7 @@ def _finalize_extensions(self) -> None:
             # Some mock objects for tests give errors
             # if we use `getattr`
             try:
-                if entry.extension.finalize:
-                    entry.extension.finalize(self)
+                entry.extension.finalize(self)
```

Comment:
Is this fix safe?

Reply:
https://github.com/pfnet/pytorch-pfn-extras/blob/e25f1a8b87397e94fdd668d049bb85d39454008b/pytorch_pfn_extras/training/extension.py#L188 seems not safe so will fix